README.md

```diff
@@ -145,6 +145,9 @@ The artifactId for FIPS builds is `AmazonCorrettoCryptoProvider-FIPS`.
 
 The classifier attribute could be set to `linux-aarch_64` to use ACCP on Linux ARM64 platforms.
 
+Starting from verions 2.3.2, the classifier attribute could be set to `osx-x86_64` to use ACCP
+on MacOS X86 platforms. The FIPS artifacts do not support MacOS X86.
```

Comment:
Can you add a note in the [FIPS section of the README](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/README.md#notes-on-accp-fips) to indicate the platforms that do or do not have a FIPS build?

Reply:
Done.